test/migrator_test.dart

```diff
@@ -16,26 +17,34 @@ import 'package:test_descriptor/test_descriptor.dart' as d;
 /// Runs all migration tests. See migrations/README.md for details.
 void main() {
   glyph.ascii = true;
-  var migrationTests = Directory("test/migrations");
-  for (var file in migrationTests.listSync().whereType<File>()) {
-    if (file.path.endsWith(".hrx")) {
-      test(p.basenameWithoutExtension(file.path), () => testHrx(file));
+  testMigrator(ModuleMigrator());
```

Comment:
I recommend creating a separate entrypoint file for each migrator you want to test, rather than putting everything in a single file. That'll make it a lot easier to just test one migrator at a time.